NightscoutUploadKit/NightscoutEntry.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  NightscoutEntry.swift
+//  RileyLink
+//
+//  Created by Timothy Mecklem on 11/5/16.
+//  Copyright © 2016 Pete Schwamb. All rights reserved.
+//
+
+import Foundation
+import MinimedKit
+
+public class NightscoutEntry : DictionaryRepresentable {
+    
```

Comment:
I would like to eventually make these all structs. But that's probably best left to a refactor.  Not requesting a change here; just a comment.